libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/daemon/MessageBusListenerActor.scala

```diff
@@ -32,9 +36,11 @@ class MessageBusListenerActor[M](source: Source[M, NotUsed])(implicit messageLik
         log.warning("Listener already subscribed. Ignoring Subscribe message")
       case Failure(ex) =>
         log.error(ex, "Source/Listener died, subscribing again")
+        monitor.onError(ex)
         trySubscribeDelayed()
```

Comment:
If this happens, than `Source` and `Sink` are stopped, unless we guarantee they are running with `Supervision.Resume`. So the  attempt to subscribe will fail. Or am I missing something?